crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__E221_E22.py.snap

```diff
@@ -106,4 +106,30 @@ expression: diagnostics
     column: 13
   fix: ~
   parent: ~
+- kind:
```

Comment:
The changes to the tests make sense to me, but I would appreciate it if someone more familiar with pycodestyle could review the changes, just to be sure.

Reply:
So I guess the difference here is that `-` is considered an operator (as in `-1`), whereas before it wasn't? And thus, `x =            -1` is considered  `MultipleSpacesBeforeOperator`?

Reply:
So I guess the difference here is that `-` is considered an operator (as in `-1`), whereas before it wasn't? And thus, `x =            -1` is considered  `MultipleSpacesBeforeOperator`?

Reply:
It'd be nice if we didn't flag this (any changes in these snapshots are deviations from pycodestyle, I think), but we could live with this if handling the `-` in `x = 1 - 2` vs. `x = -2` is hard or complicated. Either way, it'll be a lint error, it's just a question of whether it's "spaces after the `=`" or `"spaces before the `-`".

Reply:
Ah yeah, I guess pycodestyle doesn't flag `x = -   2` as a lint error at all. That seems wrong. What you have here (treating `-` as an operator everywhere) may be better anyway.

Reply:
I have a fix for the token based implementation (stack on top of this stack). I have to look closer to understand what's causing the difference in the regex based implementations (only if I can figure it out quickly)

Reply:
Ah yeah, if it's fixed upstream, then I don't feel strongly about debugging here.